templates/plugin/write_graphite/carbon.conf.erb

```diff
@@ -17,6 +17,9 @@
   LogSendErrors <%= @logsenderrors %>
   Protocol "<%= @protocol %>"
 <%- end -%>
+<%- if scope.lookupvar('collectd::collectd_version_real') and (scope.function_versioncmp([scope.lookupvar('collectd::collectd_version_real'), '5.7']) >= 0) -%>
+  PreserveSeparator <%= @preserveseparator %>
```

Comment:
Does that work or do we need to convert it do a string? Please add a unit or acceptance test for this that verifies the file content